index.js

```diff
@@ -51,10 +51,12 @@ async function fastifyOrama (fastify, options) {
     db = await Orama.create(oramaOptions)
   }
 
+  function withOrama () {
+    return this
```

Comment:
`this != oramaApi` as before - does it work?

Reply:
Yes, the idea is a little different now.
You can call `const appWithOrama = app.withOrama<Schema>()`, and then, you can access the Orama instance using `appWithOrama.orama`.
This prevents calling the getOrama every time, but you can call the `withOrama` method on top and reuse it everywhere in your plugin or file.

Reply:
That makes sense!